app/components/app_consent_component.html.erb

```diff
@@ -7,28 +7,47 @@
 <% end %>
 
 <% if patient_session.consents.present? %>
-  <% consents_grouped_by_parent.each do |summary, consents| %>
-    <%= render AppDetailsComponent.new(summary:, open: open_consents?) do %>
-      <%= render AppConsentSummaryComponent.new(**grouped_consents(consents)) %>
+  <% consents = patient_session.consents.order(:recorded_at) %>
+  <%= govuk_table do |table| %>
+    <%= table.with_head do |head| %>
+      <%= head.with_row do |row| %>
+        <%= row.with_cell(text: 'Name') %>
+        <%= row.with_cell(text: 'Response date') %>
+        <%= row.with_cell(text: 'Decision') %>
+      <% end %>
+    <% end %>
 
-      <% if consents.any?(&:response_refused?) %>
-        <p><%= contact_parent_or_guardian_link(consents) %></p>
-      <% elsif consents.any?(&:response_not_provided?) %>
-        <%= govuk_button_link_to(
-          "Get consent",
-          session_patient_manage_consent_path(
-            session,
-            patient,
-            consents.first.id,
-            :agree,
-            section: @section,
-            tab: @tab
-          ),
-          secondary: true
-        ) %>
+    <%= table.with_body do |body| %>
+      <% patient_session.consents.order(recorded_at: :desc).each do |consent| %>
```

Comment:
Since there's only ever a few consents, you can also rely on `order(:recorded_at)` being applied on the previous `consents` method and do:

```ruby
consents.reverse.each do
```

Alternatively, just not a fan of coming up with names for this:

```ruby
def consents_reversed
  patient_session.consents.order(recorded_at: :desc)
end
```

Reply:
Yup, fixed in Fixed in [b5ba1bd](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/1324/commits/b5ba1bd3a80d7293edf086cc407d10728be8b195)